<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order</class>
 <widget class="QDialog" name="order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drug Order</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Order ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Drug Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Company Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Supplier Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Order Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oid">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ocmpname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_osupname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oqnty">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ocst">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_od">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>821</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_oentry">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Order Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dentry">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Order</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Drug Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_otype">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Display/Refresh table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
